auth-web/src/components/auth/common/ContactUsTooltip.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-tooltip top max-width="450px" light content-class="tooltip" v-model="showTip">
+    <template v-slot:activator="{ attrs }">
+      <span :attrs="attrs" @click="showTip = !showTip" class="tooltip-text">Contact Us</span>
+    </template>
+    <v-card class="tooltip-content">
+      <h3 class="mb-3">Don't have a Passcode for your Cooperative Association?</h3>
+      <p>If you have no received your Access Letter from BC Registries, or have lost your Passcode, please contact us
+        at:</p>
+      <section>
+        <p class="mb-0">Toll Free: 1-877-526-1526</p>
+        <p class="mb-0">Phone: 250-387-7848</p>
```

Comment:
This is not the same phone number as in Filings UI or Create UI. Which one(s) are incorrect?

Also, you could steal some of the markup from https://github.com/bcgov/business-filings-ui/blob/master/src/components/common/ErrorContact.vue ...

Reply:
Sorry, this data was just placeholder, i forgot to specify this tooltip wasn't implemented yet. Just building the structure so when we have all the info i could update and implement. 
Should i take this out for now?

Reply:
I'm OK with leaving this code in.